test/models/prueba_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PruebaTest < ActiveSupport::TestCase
```

Comment:
One you add rules, you should add some tests like 'prueba is not valid without a nombre', 'prueba es not valid with an invalid date', 'prueba is valid with all attributers' and so on.